integration_tests/kind/metallb-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - 172.17.255.1-172.17.255.250
```

Comment:
Is this the IP range that `kind` uses? If so can we state where that's documented in a comment?

Reply:
yup, it is the IP range that `kind` will use to expose load balancer services.
I just added a comment for it.